src/Symfony/Component/Uid/UuidV1.php

```diff
@@ -45,4 +45,24 @@ public function getNode(): string
     {
         return uuid_mac($this->uid);
     }
+
+    /**
+     * @param float|null $timestamp Seconds since the Unix epoch 1970-01-01 00:00:00
+     */
+    public static function generate(float $timestamp = null, Uuid $node = null): string
```

Comment:
Should we indicate that the timestamp will be rounded to thousandths of a millisecond (same in the factory and in Uuidv6)?

Reply:
no need imho